### PR TITLE
Account for AWS change that adds region to CNAME

### PR DESCRIPTION
**Pull request by darrelldraney**
https://aws.amazon.com/releasenotes/AWS-Elastic-Beanstalk/0946276467984605

### CODE REVIEW

File: lib/eb_deployer/aws_driver/beanstalk.rb [hunk 1]
```diff
@@ -91,7 +91,8 @@ def fetch_events(app_name, env_name, params, &block)
 
       def environment_cname_prefix(app_name, env_name)
         cname = environment_cname(app_name, env_name)
-        if cname =~ /^(.+)\.elasticbeanstalk\.com/
+        region = Aws.config[:region]
+        if cname =~ /^(.+?)(\.#{region})?\.elasticbeanstalk\.com/
```
Comment (xli) [inline lib/eb_deployer/aws_driver/beanstalk.rb]: Is region from config sure to be same case with cname?

Comment (darrelldraney) [inline lib/eb_deployer/aws_driver/beanstalk.rb]: I'm fairly certain the AWS SDK requires the region to be lower case, as I get an error when I try using uppercase in the region name, but it won't hurt to add a `.downcase` to make sure. 

I see that the automated CI build failed for ruby 1.9.3. I'm not a ruby developer can you explain why? Is there something I need to fix?

**Comment by darrelldraney** [comment]
@xli is there anything I need to change so that the CI build passes for Ruby 1.9.3?

**Comment by xli** [comment]
No, something is wrong with 1.9.3 build.
But it would be great and welcome if you'd like to check it out and send another pull request to fix it

**Comment by xli** [comment]
There is conflict, could you fix it? I'll merge your change once it can be merged.

**Comment by xli** [comment]
@darrelldraney just saw @marques-work pushed a similar fix. I think he didn't see your change, so I'll close this one. Thank you for contribution, next time I will try merge it early.

**Pull request closed without merging**